deploy/csi-attacher-fcplugin.yaml

```diff
@@ -0,0 +1,60 @@
+kind: Service
```

Comment:
I don't think a service is necessary if you are not exposing any network endpoints